test/runner_test.ts

```diff
@@ -41,6 +41,13 @@ async function createServers() {
   return { httpServer, httpsServer };
 }
 
+declare global {
+  // eslint-disable-next-line no-unused-vars
+  interface Window {
```

Comment:
let's not re-define it globally? let's better add the casts inside the evaluated functions?

Reply:
Done.